CHANGELOG.md

```diff
@@ -4,6 +4,10 @@ Todos los cambios notables a este proyecto serán documentados en este archivo.
 El formato está basado en [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 y este proyecto adhiere a [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+## [1.6.1] - 2019-02-12
+### Changed
+- Las credenciales de PatPass by Webpay para ambiente de Integración han sido actualizados
```

Comment:
actualizados -> actualizadas
```suggestion
- Las credenciales de PatPass by Webpay para ambiente de Integración han sido actualizadas
```

Reply:
@leosoto great, thank you for notice, i've applied the correction. Please update your review.